Packs/ClarotyXDome/Classifiers/classifier-XDome.json

```diff
@@ -0,0 +1,21 @@
+{
+    "id": "xDome - Classifier",
+    "name": "xDome - Classifier",
+    "type": "classification",
+    "description": "Classifies xDome incidents.",
+    "keyTypeMap": {
```

Comment:
Would like to suggest mapping default to `"Custom": "xDome Custom Alert"` (or something else that can be default),so in case you will add a new type (or the existing ones will be updated), a default classification will catch it :) 
Just a suggestion.
![image](https://github.com/demisto/content/assets/88268646/7378773b-f51a-4bb0-9d6e-0a7650c7a8f2)